Lib/test/test_struct.py

```diff
@@ -578,6 +578,26 @@ def test__sizeof__(self):
         self.check_sizeof('0s', 1)
         self.check_sizeof('0c', 0)
 
+    def test_boundary_error_message(self):
+        byte_list = bytearray(1)
+        with self.assertRaisesRegex(
+                struct.error,
+                r'pack_into requires a buffer of at least 6 '
+                'bytes for packing 1 bytes at offset 5 '
+                '\(actual buffer size is 1\)'):
```

Comment:
Correct me if I am wrong (I haven’t caught up with Python after the Git Hub changeover), but I think you need to make the final part of the string a raw string:

```python-console
>>> (r'pack_into requires a buffer of at least 6 '
... 'bytes for packing 1 bytes at offset 5 '
... '\(actual buffer size is 1\)')
<stdin>:3: DeprecationWarning: invalid escape sequence \(
```